src/sentry/monitors/validators.py

```diff
@@ -258,3 +259,12 @@ def validate(self, attrs):
             del attrs["monitor_config"]
 
         return attrs
+
+
+class MonitorAlertRuleTargetValidator(serializers.Serializer):
```

Comment:
So I think you'll want to make this a sub-validator of the `MonitorValidator`. Similar to what we're doing here

https://github.com/getsentry/sentry/blob/209494fc157b6d3276afbb6d531889e7ba8061df/src/sentry/monitors/validators.py#L209

Then when the data comes in via the API we'll automatically validate this and you won't have to instantiate the validator in the endpoint at all.